src/view/RecoExplorerApp.py

```diff
@@ -39,6 +40,15 @@ def __init__(
     ) -> None:
         # basic setup
         self.config = config
+        base_url_embedding = config["ingest"]["base_url_embedding"]
```

Comment:
I feel like this whole block does not really have to do something with the Frontend/View of the application. To me it belongs on the config generation/reading side of the app.
I wouldn't expect this logic to appear here.

Reply:
Moved to RecoExplorer.py